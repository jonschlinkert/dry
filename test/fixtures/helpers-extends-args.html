{% extends "helpers-extends" %}

{% foo "two" "yes" %}
√ two:child
{% endfoo %}

{% foo "three" "yes" %}
  {% foo "four" "yes" %}
  √ three.four:child
  {% endfoo %}
{% endfoo %}