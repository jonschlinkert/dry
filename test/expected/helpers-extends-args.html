
√ one:parent

√ two:child

  
  √ three.four:child
  
